### DISCUSSION TITLE
How do I resolve "Failed to download Metals, make sure you have an internet connection and the Java Home is valid. You can configure the Java Home in the settings"?

### DISCUSSION THREAD

**Discussion by kcwill2** [scalameta/metals#2499] [Q&A]
We have mirror repositories and are using mirror.properties for Coursier config. We also tried specifying custom repos in metals config. The problem only occurs on a particular machine. It works fine everywhere else. I have confirms that the broken machine can pull deps down from the mirror repos just using sbt but I'm stumped. Is there any way to tell what repos metals is trying to use?

**Comment by tgodzik** [comment]
Did you try specifying the java home manually? Is there anything in the metals log - `Output (metals)` ?

We do have a issue open to improve a lot of things around downloading metals. Any feedback would be appreciated https://github.com/scalameta/metals/issues/2485

**Reply by kcwill2** [reply]
Thanks for responding. I tried the java_home environment variable and the metals setting. The metals.log doesn't even get created.

**Reply by tgodzik** [reply]
We might need to try and add some verbose settings to the figure out what is going on. I will try to take a look at it next week.
